.github/workflows/periodic.yml

```diff
@@ -8,6 +8,30 @@ on:
 permissions: {}
 
 jobs:
+  internal-integration:
+    name: Internal Integration
+    if: github.repository_owner == 'submariner-io'
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+      - name: Check out the repository
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8
+
+      - name: Update internal submariner-io/* dependencies to latest
+        run: |
+          go get github.com/submariner-io/admiral@devel
+          go get github.com/submariner-io/shipyard@devel
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@2b011faafdcbc9ceb11414d64d0573f37c774b04
+        with:
+          title: Update submariner-io/* dependencies to latest
+          body: To catch cross-repo integration issues before releases.
```

Comment:
The pull-request should work as a commit if merged, the body here doesn’t seem like a great commit message — it describes the purpose of the job, not the real purpose of the commit.

Reply:
What do you mean by "real purpose of the commit" vs "the purpose of the job"? All the commit does is add the job, so they have the same purpose. Happy to make better commit messages, but I thought about it a lot and I don't understand what you're looking for.

Reply:
I’m not talking about the commit message for this PR, I’m talking about the commit message generated by the job in the PRs it will create. If the goal is only to check that everything is OK, and the PRs aren’t supposed to be merged, something like

> This checks the current status of this repository against the latest version of all the Submariner project it depends on. If all goes well, this PR shouldn’t be merged. If something fails, the failure should be investigated and at least tracked as an issue (if appropriate) blocking the next release.

If the goal is to merge them,

> This pulls in the latest versions of all Submariner projects this project depends on. This is an automated update.

Reply:
Oh I see! Okay, thanks that's good feedback!

Reply:
You mentioned that you’d updated this, but I don’t see a change in the PR.